src/helm/benchmark/presentation/summarize.py

```diff
@@ -365,22 +383,25 @@ def write_cost_report(self):
             costs["total_tokens"] = costs["num_prompt_tokens"] + costs["num_completion_tokens"]
 
         write(
-            os.path.join(self.run_suite_path, "costs.json"),
+            os.path.join(self.run_release_path, "costs.json"),
             json.dumps(models_to_costs, indent=2),
         )
 
     def write_runs(self):
         write(
-            os.path.join(self.run_suite_path, "runs.json"),
+            os.path.join(self.run_release_path, "runs.json"),
             json.dumps(list(map(asdict_without_nones, self.runs)), indent=2),
         )
 
     def write_run_specs(self):
         write(
-            os.path.join(self.run_suite_path, "run_specs.json"),
+            os.path.join(self.run_release_path, "run_specs.json"),
             json.dumps(list(map(asdict_without_nones, [run.run_spec for run in self.runs])), indent=2),
         )
 
+    def write_run_manifest(self):
+        write(os.path.join(self.run_release_path, "run_manifest.json"), json.dumps(self.run_manifest, indent=2))
```

Comment:
Consider naming this `runs_to_run_suites.json` or `run_suite_mapping.json` or `run_to_run_suite.json` to make it clearer what this is? Same for variable names.